### PR TITLE
[IMP]web: pie, bar and line chart tooltip information improvements

### PR DESCRIPTION
**Pull request by jso-odoo**
Description of the issue/feature this PR addresses:
- To make tooltip information meaningful
- Related task : https://www.odoo.com/web#id=1917940&action=327&model=project.task&view_type=form&menu_id=4720
- Pad: https://pad.odoo.com/p/r.3ddff15d323dc5b772f32094f2963585

Current behavior before PR:
- Tooltip information was not meaningful in bar, line and pie chart. 

Desired behavior after PR is merged:
- Added custom template which is the same as given by the library nvD3 for the tooltip which will show the measure of the chart on the top of the tooltip in both normal and comparision view.
Tooltip data(Measure) will be changed according to the
measure of the graph and data will be accordingly to the applied groupby fields.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/js/views/graph/graph_renderer.js [hunk 1]
```diff
@@ -80,6 +82,16 @@ return AbstractRenderer.extend({
...
+        if (params.context !== undefined) {
+            var timeRangeMenuData = params.context.timeRangeMenuData;
+            if (timeRangeMenuData) {
+                this.timeRangeDescription = timeRangeMenuData.timeRangeDescription;
+                this.comparisonTimeRangeDescription = timeRangeMenuData.comparisonTimeRangeDescription;
+            } else {
+                this.timeRangeDescription = undefined;
+                this.comparisonTimeRangeDescription = undefined;
+            }
+        }
```
Comment (aab-odoo) [inline addons/web/static/src/js/views/graph/graph_renderer.js]: i'd move all this to the model directly, and store the description info into the state (alongside other timerange info)

**Comment by aab-odoo** [comment]
robodoo r+

**Comment by Polymorphe57** [comment]
robodoo retry

**Comment by aab-odoo** [comment]
robodoo retry

**Pull request closed without merging**